config/routes.rb

```diff
@@ -15,6 +15,7 @@
     patch 'users/reorder_call_list', to: 'users#reorder_call_list', as: 'reorder_call_list', defaults: { format: :js }
   end
   root :to => redirect('/users/sign_in')
+  resources :users, only: [:new, :create]
```

Comment:
For readability's sake, can we put this in the `authenticate` block if it doesn't bust up everything? (If it DOES make things break, we'll just get it later.)